### ISSUE TITLE
ReactServerAgent.del doesn't accept post args

### ISSUE THREAD

**Issue by roblg** [redfin/react-server#841]
Expected: the following should work

```
ReactServerAgent.del('/foo', {foo: "bar"}).then(() => { ... })
```

Actual: it doesn't.

I believe this works though:

```
ReactServerAgent.del('/foo').send({foo: "bar"}).then(() => { ... });
```

Looks like `ReactServerAgent.del` doesn't follow the same pattern as its siblings in allowing the shorthand. Seems like it should.

**Comment by cashhh** [comment]
I'd like to give this a shot

**Comment by drewpc** [comment]
Fixed in #882
